bundles/org.openhab.binding.mqtt.generic/src/main/resources/ESH-INF/config/color-channel-config.xml

```diff
@@ -52,6 +52,11 @@
 			<advanced>true</advanced>
 			<default>%s</default>
 		</parameter>
+		<parameter name="qos" type="integer" min="0" max="2" required="false">
```

Comment:
Does it make sense to add an options list here instead of using `min` and `max` attributes (like in https://github.com/openhab/openhab-core/blob/7ba7ee2a868f2958ad6d615dafb7f071e7df2cd6/bundles/org.openhab.core.io.transport.mqtt/src/main/resources/ESH-INF/config/brokerConnectionInstance.xml#L74) and a `default` value?

```xml
			<options>
				<option value="0">At most once (best effort delivery "fire and forget")</option>
				<option value="1">At least once (guaranteed that a message will be delivered at least once)</option>
				<option value="2">Exactly once (guarantees that each message is received only once by the counterpart)</option>
			</options>
```

Reply:
list: Good idea. 

default: I did not set a default value to keep it backward compatible, the defult is waht ever is set for the connection.